### ISSUE TITLE
live tv no longer works

### ISSUE THREAD

**Issue by cpankonien** [jellyfin/jellyfin#4973]
I have been using live tv for over a month, it mostly worked but would occaisionaly stop working for a period of time, then would start working again. 

**System (please complete the following information):**
 - OS: Ubuntu 20.04.1
 - Virtualization: none
 - Clients: Roku, browser
 - Browser: Chrome 87.0.4280.109 
 - Jellyfin Version: 10.6.4
 - Playback: Live TV 
 - Installed Plugins: AudioDB, MusicBrainz, OMDb, The TVDB
 - Reverse Proxy: none
 - Base URL: none
 - Networking: VPN
 - Storage: local

**To Reproduce**
1. Go to 'Live TV'
2. Select any channel
3. Error "Error loading channel date" "unable to load channel data from the server"

**Expected behavior**
Live tv stream starts playing

**Logs**
No errors noted in logs

**Additional context**
Tuner is set up as m3u tuner. Expected channels show up in guide. 
Guide data is xmltv from provider, expected episodes, etc appear to be in guide data.
Stream urls listed in m3u file work fine when pasted into VLC, so a network error is not suspected. 

What specifically does this error message mean? Does it mean that Jellyfin cannot get the stream url data from the .m3u file, or that it can not stream data from the stream url listed in the .m3u file?

**Comment by BaronGreenback** [comment]
Can you enable debug logging, try again, then paste the log please?

locate logging.default.json, change default to Debug as below, save, then restart Jellyfin.
```
{
    "Serilog": {
        "MinimumLevel": {
            "Default": "Debug",
```

**Comment by cpankonien** [comment]
here's what looks relevant to me (the log file is huge). If I've missed and you need the whole file i can do that, just let me know, I appreciate your response!

this part looks like it might be an issue "[2021-01-07 18:55:07.518 -05:00] [INF] Profile: "Unknown Profile", No direct play profiles found for Path: "https://vipvikingz.com/1011978582/xa2yXxb4BK/285"


[2021-01-07 18:55:07.381 -05:00] [INF] Found m3u channel: "0 group-title=\"USA NETWORK TV - A\",US: COZI TV | HD"
[2021-01-07 18:55:07.384 -05:00] [INF] Found m3u channel: "0 group-title=\"USA DOCUMENTARY\",US: DISCOVERY (EAST) | HD"
[2021-01-07 18:55:07.384 -05:00] [INF] Found m3u channel: "0 group-title=\"USA DOCUMENTARY\",US: DISCOVERY SCIENCE | HD"
[2021-01-07 18:55:07.384 -05:00] [INF] Found m3u channel: "0 group-title=\"USA SPORTS\",US: ESPN 2 | HD"
[2021-01-07 18:55:07.384 -05:00] [INF] Found m3u channel: "0 group-title=\"USA SPORTS\",US: ESPN NEWS | HD"
[2021-01-07 18:55:07.384 -05:00] [INF] Found m3u channel: "0 group-title=\"USA SPORTS\",US: ESPN | HD"
[2021-01-07 18:55:07.385 -05:00] [INF] Found m3u channel: "0 group-title=\"USA NETWORK TV - A\",US: FX (EAST) | HD"
[2021-01-07 18:55:07.385 -05:00] [INF] Found m3u channel: "0 group-title=\"USA NETWORK TV - A\",US: FXX | HD"
[2021-01-07 18:55:07.385 -05:00] [INF] Found m3u channel: "0 group-title=\"USA NETWORK TV - B\",US: FYI | HD"
[2021-01-07 18:55:07.385 -05:00] [INF] Found m3u channel: "0 group-title=\"USA NETWORK TV - B\",US: IFC | HD"
[2021-01-07 18:55:07.385 -05:00] [INF] Found m3u channel: "0 group-title=\"USA NETWORK TV - A\",US: ME TV | HD"
[2021-01-07 18:55:07.385 -05:00] [INF] Found m3u channel: "0 group-title=\"USA DOCUMENTARY\",US: NAT GEO WILD (EAST) | HD"
[2021-01-07 18:55:07.386 -05:00] [INF] Found m3u channel: "0 group-title=\"USA DOCUMENTARY\",US: NATIONAL GEOGRAPHIC (EAST) | HD"
[2021-01-07 18:55:07.386 -05:00] [INF] Found m3u channel: "0 group-title=\"USA DOCUMENTARY\",US: SMITHSONIAN CHANNEL (EAST) | HD"
[2021-01-07 18:55:07.386 -05:00] [INF] Found m3u channel: "0 group-title=\"USA NEWS\",US: THE WEATHER CHANNEL | HD"
[2021-01-07 18:55:07.386 -05:00] [INF] Found m3u channel: "0 group-title=\"USA NETWORK TV - A\",US: TLC (EAST) | HD"
[2021-01-07 18:55:07.386 -05:00] [INF] Found m3u channel: "0 group-title=\"USA NETWORK TV - A\",US: USA NETWORK (EAST) | HD"
[2021-01-07 18:55:07.387 -05:00] [INF] Found m3u channel: "0 group-title=\"USA DOCUMENTARY\",US: H2 | HD"
[2021-01-07 18:55:07.387 -05:00] [INF] Found m3u channel: "0 group-title=\"USA NETWORK TV - A\",US: DISNEY CHANNEL | SD"
[2021-01-07 18:55:07.387 -05:00] [INF] Found m3u channel: "0 group-title=\"USA NETWORK TV - A\",US: AMC | HD"
[2021-01-07 18:55:07.387 -05:00] [INF] Found m3u channel: "0 group-title=\"USA DOCUMENTARY\",US: AMERICAN HEROES CHANNEL | HD"
[2021-01-07 18:55:07.387 -05:00] [INF] Found m3u channel: "0 group-title=\"USA NETWORK TV - A\",US: BBC AMERICA | HD"
[2021-01-07 18:55:07.388 -05:00] [INF] Found m3u channel: "0 group-title=\"USA DOCUMENTARY\",US: COOKING CHANNEL | HD"
[2021-01-07 18:55:07.388 -05:00] [INF] Found m3u channel: "0 group-title=\"USA NETWORK TV - A\",US: FOOD NETWORK (EAST) | HD"
[2021-01-07 18:55:07.388 -05:00] [INF] Found m3u channel: "0 group-title=\"USA NEWS\",US: FOX NEWS CHANNEL | HD"
[2021-01-07 18:55:07.388 -05:00] [INF] Found m3u channel: "0 group-title=\"USA NETWORK TV - A\",US: HGTV (EAST) | HD"
[2021-01-07 18:55:07.388 -05:00] [INF] Found m3u channel: "0 group-title=\"USA DOCUMENTARY\",US: HISTORY CHANNEL | HD"
[2021-01-07 18:55:07.389 -05:00] [INF] Found m3u channel: "0 group-title=\"USA NETWORK TV - A\",US: SUNDANCE TV | HD"
[2021-01-07 18:55:07.389 -05:00] [INF] Found m3u channel: "0 group-title=\"USA NETWORK TV - A\",US: TV LAND (EAST) | HD"
[2021-01-07 18:55:07.389 -05:00] [INF] Found m3u channel: "0 group-title=\"USA NETWORK TV - A\",US: WE TV | HD"
[2021-01-07 18:55:07.389 -05:00] [INF] Found m3u channel: "0 group-title=\"USA NETWORK TV - A\",US: BRAVO | HD"
[2021-01-07 18:55:07.389 -05:00] [INF] Found m3u channel: "0 group-title=\"USA NETWORK TV - A\",US: TBS (EAST) | SD"
[2021-01-07 18:55:07.389 -05:00] [INF] Found m3u channel: "0 group-title=\"USA NETWORK TV - A\",US: TNT (EAST) | SD"
[2021-01-07 18:55:07.390 -05:00] [INF] Found m3u channel: "0 group-title=\"USA DOCUMENTARY\",US: HEROES AND ICONS | SD"
[2021-01-07 18:55:07.390 -05:00] [INF] Found m3u channel: "0 group-title=\"USA NETWORK TV - A\",US: DISCOVERY CHANNEL (EAST) | SD"
[2021-01-07 18:55:07.390 -05:00] [INF] Found m3u channel: "0 group-title=\"USA DOCUMENTARY\",US: SCIENCE | HD"
[2021-01-07 18:55:07.475 -05:00] [INF] User policy for "cliff". EnablePlaybackRemuxing: True EnableVideoPlaybackTranscoding: True EnableAudioPlaybackTranscoding: True
[2021-01-07 18:55:07.508 -05:00] [INF] Profile: "Unknown Profile", Path: "https://vipvikingz.com/1011978582/xa2yXxb4BK/285", isEligibleForDirectPlay: True, isEligibleForDirectStream: True
[2021-01-07 18:55:07.518 -05:00] [INF] Profile: "Unknown Profile", No direct play profiles found for Path: "https://vipvikingz.com/1011978582/xa2yXxb4BK/285"
[2021-01-07 18:55:07.547 -05:00] [INF] Profile: "Unknown Profile", Path: "https://vipvikingz.com/1011978582/xa2yXxb4BK/285", isEligibleForDirectPlay: True, isEligibleForDirectStream: True
[2021-01-07 18:55:07.547 -05:00] [INF] Profile: "Unknown Profile", No direct play profiles found for Path: "https://vipvikingz.com/1011978582/xa2yXxb4BK/285"
[2021-01-07 18:55:07.575 -05:00] [INF] Opening channel stream from "Emby", external channel Id: "m3u_7a3b02233c630c89dd8e97137577e9d879e2e7af66833c8d408d3192055ae690"
[2021-01-07 18:55:07.578 -05:00] [INF] Streaming Channel m3u_7a3b02233c630c89dd8e97137577e9d879e2e7af66833c8d408d3192055ae690
[2021-01-07 18:55:07.591 -05:00] [INF] Opening SharedHttpStream Live stream from "https://vipvikingz.com/1011978582/xa2yXxb4BK/285"
[2021-01-07 18:55:08.377 -05:00] [ERR] Error opening tuner
MediaBrowser.Model.Net.HttpException: Found
   at Emby.Server.Implementations.HttpClientManager.HttpClientManager.EnsureSuccessStatusCode(HttpResponseMessage response, HttpRequestOptions options)
   at Emby.Server.Implementations.HttpClientManager.HttpClientManager.SendAsyncInternal(HttpRequestOptions options, HttpMethod httpMethod)
   at Emby.Server.Implementations.HttpClientManager.HttpClientManager.SendAsync(HttpRequestOptions options, HttpMethod httpMethod)
   at Emby.Server.Implementations.LiveTv.TunerHosts.SharedHttpStream.Open(CancellationToken openCancellationToken)
   at Emby.Server.Implementations.LiveTv.TunerHosts.BaseTunerHost.GetChannelStream(String channelId, String streamId, List`1 currentLiveStreams, CancellationToken cancellationToken)
[2021-01-07 18:55:08.393 -05:00] [ERR] Error processing request. URL: "http://192.168.50.5:8096/Items/131000e73745774b51d7eef7e8d6cf59/PlaybackInfo?AutoOpenLiveStream=true&IsPlayback=true&MaxStreamingBitrate=140000000&StartTimeTicks=0&UserId=0c2fb53eec4b4a6f8a4f5c64f7b65541"
MediaBrowser.Controller.LiveTv.LiveTvConflictException: Exception of type 'MediaBrowser.Controller.LiveTv.LiveTvConflictException' was thrown.
   at Emby.Server.Implementations.LiveTv.TunerHosts.BaseTunerHost.GetChannelStream(String channelId, String streamId, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.EmbyTV.EmbyTV.GetChannelStreamWithDirectStreamProvider(String channelId, String streamId, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.LiveTvManager.GetChannelStream(String id, String mediaSourceId, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.LiveTv.LiveTvMediaSourceProvider.OpenMediaSource(String openToken, List`1 currentLiveStreams, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Library.MediaSourceManager.OpenLiveStreamInternal(LiveStreamRequest request, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Library.MediaSourceManager.OpenLiveStream(LiveStreamRequest request, CancellationToken cancellationToken)
   at MediaBrowser.Api.Playback.MediaInfoService.OpenMediaSource(OpenMediaSource request)
   at MediaBrowser.Api.Playback.MediaInfoService.GetPlaybackInfo(GetPostedPlaybackInfo request)
   at MediaBrowser.Api.Playback.MediaInfoService.Post(GetPostedPlaybackInfo request)
   at Emby.Server.Implementations.Services.ServiceExecGeneral.GetTaskResult(Task task)
   at Emby.Server.Implementations.Services.ServiceHandler.ProcessRequestAsync(HttpListenerHost httpHost, IRequest httpReq, HttpResponse httpRes, ILogger logger, CancellationToken cancellationToken)
   at Emby.Server.Implementations.HttpServer.HttpListenerHost.RequestHandler(IHttpRequest httpReq, String urlString, String host, String localPath, CancellationToken cancellationToken)
[2021-01-07 18:55:08.396 -05:00] [WRN] HTTP Response 500 to "192.168.50.32". Time (slow): 0:00:01.1110289. "http://192.168.50.5:8096/Items/131000e73745774b51d7eef7e8d6cf59/PlaybackInfo?AutoOpenLiveStream=true&IsPlayback=true&MaxStreamingBitrate=140000000&StartTimeTicks=0&UserId=0c2fb53eec4b4a6f8a4f5c64f7b65541"
[2021-01-07 18:55:12.160 -05:00] [ERR] Error processing request. URL: "http://192.168.50.5:8096/Items/e1a04ac652d423646a90eeaf7885fc44/Images/Backdrop?maxHeight=261&maxWidth=464&quality=90&Tag=null&UnplayedCount=3"
MediaBrowser.Common.Extensions.ResourceNotFoundException: Rise Of Empires does not have an image of type Backdrop
   at MediaBrowser.Api.Images.ImageService.GetImage(ImageRequest request, Guid itemId, BaseItem item, Boolean isHeadRequest)
   at Emby.Server.Implementations.Services.ServiceExecGeneral.Execute(Type serviceType, IRequest request, Object instance, Object requestDto, String requestName)
   at Emby.Server.Implementations.Services.ServiceController.Execute(HttpListenerHost httpHost, Object requestDto,

**Comment by BaronGreenback** [comment]
[2021-01-07 18:55:08.377 -05:00] [ERR] Error opening tuner
MediaBrowser.Model.Net.HttpException: Found

It seems to be having a problem connecting outbound - and is returning a really useful error message.

The good news is that this http subsystem was one of the things reworked in 10.7, so it's likely that an actual error message might be logged, (that is if whatever it is causing your problem hasn't been fixed).

We're currently working on RC3 if you don't mind trying pre-release version (back up your data beforehand).

**Comment by crobibero** [comment]
From my testing, live tv that returns a redirect now works in 10.7

**Comment by cpankonien** [comment]
I'll be glad to, but I am not that familiar with installing things via github, I installed it from the ubuntu repositories. If you can give me a hint I'll get right on it. I can compile from source if necessary, but I haven't done that in 20 years or so, so I might be a bit rusty!

**Comment by cpankonien** [comment]
I've found rc2, will that suffice?

**Comment by cpankonien** [comment]
well, that fixed it! live tv now working with 10.7rc2, thanks so much!

**Comment by cpankonien** [comment]
in case anyone else needs this, I downloaded a .deb of 10.7rc2 from [here](https://repo.jellyfin.org/releases/server/debian/stable/)
